plugins.json

```
@@ -34641,6 +34641,140 @@
+    "download_url": "https://plugins.releases.mattermost.com/release/mattermost-plugin-mscalendar-v1.5.0.tar.gz",
+    "release_notes_url": "https://mattermost.com/pl/mattermost-plugin-mscalendarreleases/tag/v1.5.0",
```

```suggestion
    "release_notes_url": "https://mattermost.com/pl/mattermost-plugin-mscalendar/releases/tag/v1.5.0",
```

Let's fix it manually here, and the manifest builder code can be fixed later.  This has been broken for a year.  Not sure how/why it was not noticed - I guess BS has been dealing with it manually.

---

@marianunez @nevyangelova I had Claude fix this in all the repos:

1. mattermost-plugin-gitlab - https://github.com/mattermost/mattermost-plugin-gitlab/pull/625
2. mattermost-plugin-zoom - https://github.com/mattermost/mattermost-plugin-zoom/pull/440
3. mattermost-plugin-confluence - https://github.com/mattermost/mattermost-plugin-confluence/pull/207
4. mattermost-plugin-github - https://github.com/mattermost/mattermost-plugin-github/pull/946
5. mattermost-plugin-jira - https://github.com/mattermost/mattermost-plugin-jira/pull/1251
6. mattermost-plugin-servicenow - https://github.com/mattermost/mattermost-plugin-servicenow/pull/212
7. mattermost-plugin-mscalendar - https://github.com/mattermost/mattermost-plugin-mscalendar/pull/471
8. mattermost-plugin-google-calendar - https://github.com/mattermost/mattermost-plugin-google-calendar/pull/113